server.js

```diff
@@ -187,56 +160,68 @@ app.use([
 app.use('/api', require('./controllers/api'));
 app.use('/tools', require('./controllers/tools'));
 app.use('/patterns', require('./controllers/pattern-library'));
+app.use('/search', require('./controllers/search'));
 
 /**
- * Handle Aliases
- */
-aliases.forEach((redirect) => {
-    app.get(redirect.from, (req, res) => res.redirect(301, redirect.to));
-});
-
-/**
- * Handle legacy programme pages as wildcards
- * (eg. redirect them to /funding/programmes/<slug>)
- */
-app.get('/:region?/global-content/programmes/:country/:slug', (req, res) => {
-    const locale = req.params.region === 'welsh' ? '/welsh' : '';
-    res.redirect(301, `${locale}/funding/programmes/${req.params.slug}`);
-});
-
-/**
- * Handle migrated A Better Start child pages as wildcards
- * (eg. redirect them to /funding/publications/a-better-start/<slug>)
- */
-app.get('/funding/strategic-investments/a-better-start/:slug', (req, res) => {
-    res.redirect(
-        301,
-        `/funding/publications/a-better-start/${req.params.slug}`
-    );
-});
-
-/**
- * Archived paths
- * Handles archived pages (eg. redirect to National Archives)
- * and legacy files (eg. show a message about removed documents)
+ * Handle archived paths first:
+ * - Legacy redirects and aliases
+ * - National Archives content
+ * - Archived media
  */
 app.use('/', require('./controllers/archived'));
 
 /**
- * Initialise section routes
- * - Creates a new router for each section
- * - Apply shared middleware
- * - Apply section specific controller logic
- * - Add common routing (for static/fully-CMS powered pages)
+ * Initialise section routers
  */
-forEach(routes, function (section, sectionId) {
-    const router = express.Router();
```

Comment:
This is the big change, the "routes" definition is now a flatter list of sections and routers. Jobs and data are still top level routers, but that's just a sign that we need to move them later.

Reply:
Probably Contact too? Ideally we'd have no "top level" pages except the homepage. Could probably live in About?